<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>729</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Happiness Meter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="GBOverall">
        <property name="title">
         <string>Overall</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QRadioButton" name="RBOverallHappy">
           <property name="text">
            <string>I feel rather happy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="RBOverallNotHappy">
           <property name="text">
            <string>I don't feel quite happy</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="GBPersonalLife">
        <property name="title">
         <string>Personal Life</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="RBPersonalLTR">
           <property name="text">
            <string>I am in relations</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="RBPersonalSometimes">
           <property name="text">
            <string>I am not in relations right now</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="GBProfessionalLife">
        <property name="title">
         <string>Professional life</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QRadioButton" name="RBProfessionalBusy">
           <property name="text">
            <string>I am professionally busy (I actively earn money)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="RBProfessionalNotBusy">
           <property name="text">
            <string>I do not actively earn money</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CBEnoughMoney">
        <property name="text">
         <string>I have enough money</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="GBHobby">
        <property name="title">
         <string>Hobby</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="0">
          <widget class="QRadioButton" name="RBHobbyBad">
           <property name="text">
            <string>I do not have time to do what I like</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="RBHobbyExc">
           <property name="text">
            <string>I am spending lots of time on my hobby</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="RBHobbyNorm">
           <property name="text">
            <string>I wish I had more time to sepnd on my hobby</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="GBHealth">
        <property name="title">
         <string>Body/health</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QRadioButton" name="RBHealthExc">
           <property name="text">
            <string>I feel young, strong and beautiful</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="RBHealthNorm">
           <property name="text">
            <string>I wish I spent more time on body and health</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="RBHealthBad">
           <property name="text">
            <string>I am not happy they way I look</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="PBTell">
          <property name="text">
           <string>Tell me!</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>385</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
